Broken adm GAS transfers

## Expected Behavior
dev-env starting, tests running.

## Current Behavior
https://github.com/nspcc-dev/neofs-node/actions/runs/7315754568/job/19929720187
```
Stage 6: register candidates.
Waiting for transactions to persist...
Stage 7: transfer NEO to alphabet contracts.
Waiting for transactions to persist...
Transfer GAS to wallet ./services/storage/wallet01.json
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/nspcc-dev/neofs-node/cmd/neofs-adm/internal/modules/morph.glob..func6(0x16aa300?, {0xe7029b?, 0x8?, 0x8?})
	github.com/nspcc-dev/neofs-node/cmd/neofs-adm/internal/modules/morph/root.go:138 +0x345
github.com/spf13/cobra.(*Command).execute(0x16aa300, {0xc000140180, 0x8, 0x8})
	github.com/spf13/cobra@v1.7.0/command.go:940 +0x862
github.com/spf13/cobra.(*Command).ExecuteC(0x16a60e0)
	github.com/spf13/cobra@v1.7.0/command.go:1068 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.7.0/command.go:992
github.com/nspcc-dev/neofs-node/cmd/neofs-adm/internal/modules.Execute(...)
	github.com/nspcc-dev/neofs-node/cmd/neofs-adm/internal/modules/root.go:49
main.main()
	github.com/nspcc-dev/neofs-node/cmd/neofs-adm/main.go:10 +0x25
: Failed to transfer GAS to alphabet wallets
make[1]: *** [Makefile:117: up/bootstrap] Error 1
```

## Possible Solution
Fix the damn thing!

## Steps to Reproduce (for bugs)
Setup af001f8052a203eab408af2bf3a41c7e5af2ac11 dev-env.

## Context
We're completely screwed with no tests.

## Regression
Likely, #2690.

--- a/cmd/neofs-adm/internal/modules/morph/root.go
+++ b/cmd/neofs-adm/internal/modules/morph/root.go
@@ -130,7 +130,7 @@ var (
 					return fmt.Errorf("missing wallet path (use '--%s <out.json>')", storageWalletFlag)
 				}
 
-				w, err := wallet.NewWallet(storageWalletPath)
+				w, err := wallet.NewWalletFromFile(storageWalletPath)
 				if err != nil {
 					return fmt.Errorf("can't open wallet: %w", err)
 				}

